exercises/secret-handshake/makefile

```diff
@@ -1,3 +1,7 @@
+### If you wish to use extra libraries (math.h for instance),
+### add their flags here (-lm in our case) in the "LIBS" variable.
```

Comment:
My only nitpick is that we include `(-lm in our case)` even when we don't have `-lm` (or any `LIBS` variable at all).